Description/ru-RU.md

```diff
@@ -0,0 +1,100 @@
+
+### Основан на Fabric
+
+В сравнении с Forge, Fabric обновляется и загружается быстрее, не включает в себя громоздкие библиотеки, и всё-ещё имеет отличную поддержку модов. [(Да, Quilt на рассмотрении.)][9]
+
+### Дружелюбное сообщество
+
+Мы - растущее дружелюбное сообщество, любим обсуждать модпак и помогать друг другу. [Присоединяйся!][10]
+
+## ЧаВО
+
+> **Вопрос**: Как это установить? Обновить? Добавить моды? Или что-нибудь ещё?
+> 
+> **Ответ**: [Все ответы на нашей wiki.][11]
+
+
+> **Вопрос**: Что-то сломалось! Вы должны что-то добавить!
+> 
+> **Ответ**: [Опишите это на нашем GitHub.][8]
+
+
+> **Вопрос**: Какие моды тут есть? 
+> 
+> **Ответ**: Нажмите на любую версию и промотайте вниз [или взгляните на нашу табличку сравнения][12].
+
+
+> **Вопрос**: Я должен устанавливать этот модпак на сервер?
+> 
+> **Ответ**: Нет, он работает на всех серверах разрешающих стандартную игру. [Расширенный ответ и советы по серверу тут.][13] 
+> 
+> [![](https://i.ibb.co/gr9mSxW/image.png) Если вам нужен хостинг, жмите сюда - 25% скидка на первый месяц!][14]
+
+
+> **Вопрос**: У меня ещё вопросы!
+> 
+> **Ответ**: [Тогда добро пожаловать на наш дискорд!][10]
+
+<details>
+   <summary>Дисклеймеры</summary>
+   
+* Этот модпак должен соответствовать с правилами большинства сторонних серверов, но на вашей совести удостоверится в том разрешён он на вашем сервере или нет. Устанавливая этот модпак вы соглашаетесь с тем что его автор, разработчики модов, и Mojang не дают никаких гарантий на использование этого модпака, всё что вы с ним делаете, вы делаете на свой страх и риск. 
+   
+    * Команда */ghost* (представленная модом [AntiGhost][15]) может быть запрещена на некоторых серверах [так-как она отправляет достаточно большое количество пакетов][16], но это не должно быть проблемой пока вы ей не спамите, используйте её только если застряли в блоках.
+* Этот модпак использует [Starlight][17] для оптимизации чанков, и [в соответствии с тем как он работает][18], отключение (или замена) мода, приведёт к тому что ваши существующие миры будут загружаться немного медленнее в первый раз. Это безвредно, и значит лишь то что игре придётся заново просчитывать освещённые участки, но с другим алгоритмом. 
+* Этот модпак отключает [предупреждение о сторонних серверах][19], поэтому если вы используете сетевую игру, вы так-же соглашаетесь с данным заявлением: 
+    > Предупреждение: Сетевая игра, предложенная сторонними серверами, не является собственностью, не управляется и не модерируется Mojang Studios. Во время сетевой игры вы можете столкнуться с немодерируемыми сообщениями в чате или иными видами пользовательского контента, который может подходить не всем игрокам. 
+
+    * Вы также должны помнить о том что вы можете использовать [экран "социальных взаимодействий"][20] по нажатию на кнопку `P` (англ) в игре.
```

Comment:
Does it not adapt to the Russian keyboard layout automatically?

Reply:
Hmm. Seems like it. But I hate it, it's easier to navigate with English keybinds.
```suggestion
    * Вы также должны помнить о том что вы можете использовать [экран "социальных взаимодействий"][20] по нажатию на кнопку `З` в игре.
```